### PR TITLE
[BLUEBUTTON-177] Migrate to nginx and uwsgi

### PR DESCRIPTION
**Pull request by rnagle**
What's included:

- Added `roles/nginx` with installation and configuration of nginx and uwsgi
- Removed several roles related to apache and mod_wsgi that we don't need anymore
- Moved new relic role up to `roles/new_relic`, beginning an effort to flatter/simplify the directory structure of the repo

Todo:
- Fix the log formatting to include same info that was in apache logs
- Update any splunk search configurations that depend on apache logs
- Test TLS configuration by deploying to `dev`, update nginx conf as necessary

### CODE REVIEW

File: roles/nginx/files/nginx.conf [hunk 1]
```diff
@@ -0,0 +1,63 @@
+user nginx nginx;
+worker_processes 2;
```
Comment (whytheplatypus) [inline roles/nginx/files/nginx.conf:2]: I'd be interested to see what the `auto` setting does, but maybe that's a bad idea.

**Review by whytheplatypus — APPROVED** [review]

**Review by dtisza1 — COMMENTED** [review]
For Splunk ingestion, the permissions on the /var/log/nginx directory need updated for "rx" group bits and group = "logreader".

**Review by dtisza1 — APPROVED** [review]
Looks good.

**Review by whytheplatypus — APPROVED** [review]
lgtm

**Pull request merged**